internal/utils/utils.go

```diff
@@ -240,3 +241,13 @@ func ParseErrorResponse(errorMsg string) (*CustomError, error) {
 	}
 	return &customErr, nil
 }
+
+// ValidateUUID checks if a given string is a UUID or not
+func ValidateUUID(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
I don't understand this function signature. Why are v and ws necessary? 
We could accomplish the same with k and returning errors.

Reply:
is it because the validate func expects that signature?

Reply:
(answer was yes)